offerzen/global/company-fast-track-access/company-prospect-form@v1.2.js

```diff
@@ -0,0 +1,241 @@
+(function () {
+  // FORM TARGETTING
+  const prospectFormId = '#Company-Prospect-Form';
+  const pageRef = 'onSubmitCompanyProspectForm';
+  const formSubmitButtonSelector = '#submit-button';
+  const formContainerClass = '.js-company-prospect-form';
+  // END FORM TARGETING
+  // This needs to run regardless of whether JQuery has loaded
+  // Disable button
+  const formSubmitButton = document.querySelector(
+    `${prospectFormId} ${formSubmitButtonSelector}`
+  );
+
+  // Store button labels for state changes
+  let initialButtonValue = formSubmitButton.getAttribute('data-initial');
+  let dataWait = formSubmitButton.getAttribute('data-wait');
+  let dataBusy = formSubmitButton.getAttribute('data-busy');
+
+  function disableSubmitButton(buttonLabel) {
+    formSubmitButton.setAttribute('disabled', 'disabled');
+    formSubmitButton.value = buttonLabel;
+  }
+
+  function enableSubmitButton() {
+    formSubmitButton.removeAttribute('disabled');
+    formSubmitButton.value = initialButtonValue;
+  }
+
+  disableSubmitButton('Loading...');
+  // ------------------------------------------------------------------------------------------
+  window.$loaded(function () {
+    window.$parsleyLoaded = function (cb) {
+      setTimeout(() => {
+        if (window.Parsley && window.ParsleyValidator) {
+          cb(window, document, parsley, undefined);
+          return;
+        }
+        $parsleyLoaded(cb);
+      }, 50);
+    };
+
+    window.$parsleyLoaded(function (window, document, parsley) {
+      const form = $(prospectFormId);
+      const formContainer = $(formContainerClass);
+      const urlParams = new URLSearchParams(window.location.search);
+      let option = '';
+      if (urlParams.has('option')) {
+        option = urlParams.get('option');
+      }
```

Comment:
Check for option param so option variable can be available for both `onSubmitForm` and `startProspectPolling` functions